src/app/data-model-provider/ProviderMetadataTree.h

```diff
@@ -82,6 +103,11 @@ class ProviderMetadataTree
     /// the attribute changes.
     virtual void Temporary_ReportAttributeChanged(const AttributePathParams & path) = 0;
 
+    virtual void SetNodeConfigurationListener(NodeConfigurationListener * listener)                           = 0;
+    virtual void NotifyNodeConfigurationListener()                                                            = 0;
+    virtual CHIP_ERROR GetNodeDataModelConfiguration(NodeDataModelConfiguration & nodeDataModelConfiguration) = 0;
```

Comment:
Again: in the multiplexing case, I am not sure how this would get implemented on either the thing doing the multiplexing or the things being multiplexed.